test/controllers/api/release_controller_test.exs

```diff
@@ -87,7 +87,7 @@ defmodule HexWeb.API.ReleaseControllerTest do
     assert conn.status == 422
     body = Poison.decode!(conn.resp_body)
     assert body["message"] == "Validation error(s)"
-    assert body["errors"]["meta"]["links"] == "expected type dict(string, string)"
+    assert body["errors"]["meta"]["links"] == "is invalid"
```

Comment:
We can leave as-is for now, but this could be useful to have in ecto. I see no reason why ecto can't include the expected type.

@josevalim Thoughts?